src/Form.php

```diff
@@ -595,7 +595,7 @@ public function ajaxSubmit()
                 ob_start();
                 $this->loadPOST();
                 $response = $this->hook('submit');
-                $output = ob_get_contents();
+                $output = ob_get_clean();
```

Comment:
i raise exception to the upper level to be caught by the global exception handler